src/bin/nydus-image/main.rs

```diff
@@ -1038,6 +1098,49 @@ impl Command {
         OutputSerializer::dump(matches, build_output, build_info)
     }
 
+    fn chunkdict_save(matches: &ArgMatches) -> Result<()> {
+        let bootstrap_path = Self::get_bootstrap(matches)?;
+        let config = Self::get_configuration(matches)?;
+        let db_path = matches.get_one::<String>("database-path").unwrap();
+        let db_type = matches.get_one::<String>("database-type").unwrap();
+        debug!("db_path: {}", db_path);
+        debug!("db_type: {}", db_type);
+        // For backward compatibility with v2.1
+        config
+            .internal
+            .set_blob_accessible(matches.get_one::<String>("bootstrap").is_none());
+
+        let blobs: Vec<Arc<nydus_storage::device::BlobInfo>> = match db_type.as_str() {
+            "Sqlite" => {
+                let mut deduplicate =
+                    Deduplicate::<SqliteDatabase>::new(bootstrap_path, config, db_path)?;
+                deduplicate
+                    .save_metadata(Some(&db_path))
+                    .with_context(|| format!("failed to check bootstrap {:?}", bootstrap_path))?
+            }
+            // Add other cases for different database types if needed
+            _ => {
+                return Err(anyhow!("Unsupported database type"));
```

Comment:
Should give the type name in the error message.

Reply:
Improved prompt messages。